plugin.json

```diff
@@ -1,12 +1,12 @@
 {
     "id": "mattermost-plugin-servicenow-virtual-agent",
     "name": "ServiceNow Virtual Agent",
+    "description": "This plugin is for integrating Mattermost with ServiceNow Virtual Agent.",
     "homepage_url": "https://github.com/mattermost/mattermost-plugin-servicenow-virtual-agent",
     "support_url": "https://github.com/mattermost/mattermost-plugin-servicenow-virtual-agent/issues",
-    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-servicenow-virtual-agent/releases/tag/v2.1.1",
-    "description": "This plugin is for integrating Mattermost with ServiceNow Virtual Agent.",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-servicenow-virtual-agent/releases/tag/v2.1.2",
     "icon_path": "assets/starter-template-icon.svg",
-    "version": "2.1.1",
+    "version": "2.1.2",
     "min_server_version": "5.37.0",
```

Comment:
@hanzei, is the min server version correct/realistic?

Reply:
@manojmalik20 DId you test the plugin against a 5.37.0 server?

Reply:
@hanzei Yes, we've tested the plugin against a 5.37.0 server.